### PR TITLE
ctDNA Configurations

### PR DESCRIPTION
**Pull request by SophieS9**
- Addition of pipeline script to upload the results for ctDNA. Largely reflects the process currently used for TSO500. 
- Unit tests and accompanying data added for ctDNA - all unit tests run successfully.

Tested on runs 230406_A00748_0373_AHJVTYDMXY and 230125_A01771_0151_AHC5LCDMXY on hard drive under AutoQC_ctDNA/.

### CODE REVIEW

File: qc_database/management/commands/update_database.py [hunk 1]
```diff
@@ -1245,4 +1253,109 @@ def handle(self, *args, **options):
...
+
+					run_id=run_analysis.run.run_id
+					run_config_key = run_analysis.pipeline.pipeline_id + '-' + run_analysis.analysis_type.analysis_type_id
+
+					if run_config_key not in config_dict['pipelines']:
+
+						ctDNA = ctDNA_pipeline.TSO500_ctDNA(results_dir = run_data_dir,
+															sample_names = sample_ids,
+															run_id = run_analysis.run.run_id,
+															sample_completed_files=['*_fusion_check.csv', '*_variants.tsv', '*_coverage.json'],
```
Comment (erikwaskiewicz) [inline qc_database/management/commands/update_database.py:1269]: Does this mean there must be a `*_fusion_check.csv` file for AutoQC to update? Just thinking that there might be panels where fusions arent required (like the RAS one), so maybe we should remove this

File: qc_database/tests/test_pipelines.py [hunk 2]
```diff
@@ -298,5 +298,62 @@ def test_tso500(self):
 			#run valid
 			run_valid= tso500.run_is_valid()
 			self.assertEqual(run_valid, False)
+			
+	def test_ctDNA(self):
+
+
+			results_dir = 'test_data/ctDNA_test'
+			ctDNA = ctDNA_pipeline.TSO500_ctDNA(results_dir = 'test_data/ctDNA_test/',
+																sample_completed_files= ['*_fusion_check.csv', '*_variants.tsv', '*_coverage.json'],
```
Comment (erikwaskiewicz) [inline qc_database/tests/test_pipelines.py:307]: Same as comment above

**Review by erikwaskiewicz — COMMENTED** [review]
Code looks good, one comment that I think we need to remove the fusions file from the required outputs, otherwise all good. 

I still need to install locally and run unit tests before signing off

File: qc_database/tests/test_pipelines.py (hunk 2 above)
Comment (SophieS9) [reply qc_database/tests/test_pipelines.py:307]: As above

**Comment by Istaisa** [comment]
Unit tests run locally, all pass.

**Pull request merged**